src/renderer/OpenGL.zig

```diff
@@ -2344,9 +2344,9 @@ fn drawCellProgram(
 
     // Clear the surface
     gl.clearColor(
-        @as(f32, @floatFromInt(self.draw_background.r)) / 255,
-        @as(f32, @floatFromInt(self.draw_background.g)) / 255,
-        @as(f32, @floatFromInt(self.draw_background.b)) / 255,
+        @as(f32, @floatFromInt(self.draw_background.r)) / 255 * self.config.background_opacity,
+        @as(f32, @floatFromInt(self.draw_background.g)) / 255 * self.config.background_opacity,
+        @as(f32, @floatFromInt(self.draw_background.b)) / 255 * self.config.background_opacity,
```

Comment:
Had some typing issues when trying to compile this on Linux, it expected an `f32` when an `f64` was being produced by the expression, fixed it with a `floatCast`
```suggestion
        @floatCast(@as(f32, @floatFromInt(self.draw_background.r)) / 255 * self.config.background_opacity),
        @floatCast(@as(f32, @floatFromInt(self.draw_background.g)) / 255 * self.config.background_opacity),
        @floatCast(@as(f32, @floatFromInt(self.draw_background.b)) / 255 * self.config.background_opacity),
```

Reply:
Thanks!